Makefile

```diff
@@ -82,7 +82,19 @@ vagrant-deploy: vagrant-sync-config vagrant-sync-build
 .release-functest:
 	make functest > .release-functest 2>&1
 
+kubevirt.yaml: TAG=$(shell git describe) PREFIX=kubevirt
+kubevirt.yaml:
+	echo "docker_prefix=$(PREFIX)" > hack/config-local.sh
+	echo "docker_tag=$(TAG)" >> hack/config-local.sh
+	./hack/build-manifests.sh
+	echo "# Created on $(shell date) at $(shell git describe --always)" > $@
+	echo "# For tag '$(TAG)'" >> $@
+	for M in manifests/*.yaml; do cat $$M ; echo "---" ; done >> $@
```

Comment:
There is one catch, the dependency order of the yaml files. The resources and the system users should go first. Then the rest. I know that the kubernetes will handle it at some point, but it makes the whole stuff more readable.

Reply:
I just to prior art and reused it, but yes, that is something to consider